### PR TITLE
Marketplace page

### PR DESCRIPTION
**Pull request by rssilva**
- [ ] Font SFProText, SFUIText
- [x] Search Icon
- [x] Modal close icon
- [x] Apps icon
- [x] Apps JSON from API
- [x] Download Link
- [x] Copy Url

EDITED: removed the old prints since we have a brand new one

### CODE REVIEW

**Comment by isabellarussell** [comment]
@rssilva can we change the position of the rocket? It interferes with the page title
https://user-images.githubusercontent.com/2047941/42511647-db15dbec-8428-11e8-89c4-f808a1483470.png

Also, will this be the final image or is it a placeholder? I thought we were going for a loading rocket idea?

**Comment by rssilva** [comment]
Oh, sorry @isabellarussell. Totally forgot to remove the placeholder image with the right one

screen shot 2018-07-11 at 9 00 39 am

File: apps.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+[
```
Comment (graywolf336) [inline apps.json]: This file can be removed now, right?

File: marketplace.js [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+
+    return parsed
+  }
+
+  var findAppByName = function (name, apps) {
+    var app = {}
+
+    for (var i = 0; i < apps.length; i++) {
+      if (apps[i].name === name) {
+        app = apps[i]
```
Comment (graywolf336) [inline marketplace.js]: You can return out of here, that way the loop stops as soon as it finds the app

**Comment by vynmera** [comment]
How will Developers submit Apps to the Platform?

**Comment by geekgonecrazy** [comment]
@vynmera https://forums.rocket.chat/t/submitting-your-rocket-chat-apps/1848

**Pull request merged**